modules/sdk-coin-btc/test/unit/recovery/backupKeyRecovery.ts

```diff
@@ -0,0 +1,298 @@
+/**
+ * @prettier
+ */
+import 'should';
+import sinon from 'sinon';
```

Comment:
Should we add `sinon`, `nock`, and `bip32` in devDependencies? Also, `lodash` and `should` are used in other test files.

Reply:
Our shared devDependencies between multiple modules are hoisted to the root of the monorepo.

Reply:
I saw in some other coin packages we added those test dependent packages to devDependencies so that's why I wanted to make sure what we should do here as we refactor more coins.